src/components/SpaceBg/index.js

```diff
@@ -0,0 +1,266 @@
+import React from 'react'
+import styled, { keyframes } from 'styled-components'
+
+import HeroImage from '../../assets/images/hero-graphic-desktop.png'
+import HeroImageLeft from '../../assets/images/hero-graphic-left.png'
+import { breakpoints, gradients } from '../../utils/theme'
+
+export const SpaceBg = ({ children }) => (
+  <StyledHero id="liquidity-hero" className="hero-active">
+    <div className="inner-hero">
+      <AppBodyContainer>{children}</AppBodyContainer>
+      <div className="hero-background">
+        <div className="hero-image hero-image-right"></div>
+        <div className="hero-image hero-image-left"></div>
+      </div>
+    </div>
+  </StyledHero>
+)
+
+const AppBodyContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  z-index: 3;
+  min-height: calc(100vh - 340px);
+`
+
+const zoomOut = keyframes`
+    0% {transform: scale(1.2); opacity: 0;}
+    100% {transform: scale(1); opacity: 0.7;}
+`
+
+const StyledHero = styled.div`
```

Comment:
Is there a way to not repeat this code? If think the same style is being used in the swap page / landing

Reply:
This should apply to other repeating code in this file

Reply:
We should the same Styled hero and background image but I think we should do it after nailing the right abstractions. They are currently different as the hero on swap page has a lot of other things.